goss-testing/tests/ncn/goss-bgp-established-mellanox.yaml

```diff
@@ -0,0 +1,43 @@
+#
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# Test requirements: sshpass must be installed and the SSHPASS environment
+# variable must be set to the admin user password of the spine switches.
+
+{{ $scripts := .Env.GOSS_BASE | printf "%s/scripts" }}
+{{ $logrun := $scripts | printf "%s/log_run.sh" }}
+command:
+    {{ $testlabel := "bgp-mellanox-status" }}
+    {{$testlabel}}:
+        title: Check if bgp mellanox status
+        meta:
+            desc: If this test fails, make sure system is using Mellanox switch. See troubleshooting/known_issues/check_system_management_monitoring_tools.md in the CSM documentation for further remediation steps
+            sev: 0
+        exec: |-
+            #Set pipefail so that if any commands in the command pipeline fail, the test will fail
+            set -eo pipefail
+            "{{$logrun}}" -l "{{$testlabel}}" \
+                sshpass -e ssh admin@sw-spine-001.hmn cli \"enable\" \"show ip bgp vrf all summary\"|grep -i established
```

Comment:
Also, is it intended to only check one of the spine switches? I thought our BGP tests usually check both, since problems can be seen on one but not the other.